rust/core-lib/src/event_context.rs

```diff
@@ -195,8 +195,15 @@ impl<'a> EventContext<'a> {
                 // The action that follows the block must belong to a separate undo group
                 self.editor.borrow_mut().update_edit_type();
 
-                let delta = self.editor.borrow_mut().delta_rev_head(starting_revision);
-                self.update_plugins(&mut self.editor.borrow_mut(), delta, "core")
+                let res = self.editor.borrow_mut().delta_rev_head(starting_revision);
```

Comment:
I think unwrap here is actually fine; this revision comes from core (we stash it earlier in this fn) so if we can't find it something is seriously wrong.